packages/smart-account/src/SmartAccount.ts

```diff
@@ -452,18 +452,18 @@ class SmartAccount extends EventEmitter {
       fallbackUserOp.callData = txnData
       // TODO: check if callGasLimit is valid
     }
-    try {
-      const callGasLimit = await this.transactionManager.estimateGasUsed(
-        this.address,
-        fallbackUserOp.callData,
-        chainId
-      )
-      fallbackUserOp.callGasLimit = BigNumber.from(callGasLimit)
-      this._logMessage('callGasLimit')
-      this._logMessage(callGasLimit)
-    } catch (error) {
-      this._logMessage('Error in estimating callGasLimit')
-    }
+    // try {
```

Comment:
I feel this should not get commented out. I will spend some time to check entire flow post merge